src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2888,8 +2888,8 @@ public void setStereoscopy(String stereoscopy) {
 		SBSL,
 		SBSR,
 		HSBSL,
-		OUL,
-		OUR,
+		ABL,
+		ABR,
```

Comment:
@Nadahar yes it is `OUx` but the FFmpeg filters accept the following options:
```
abl       above-below (left eye above, right eye below) 
abr       above-below (right eye above, left eye below) 
```
and I use them and not convert `OUx` to this convention.